src/gc/unix/gcenv.unix.cpp

```diff
@@ -530,6 +532,9 @@ uint32_t GCToOSInterface::GetCurrentProcessCpuCount()
     if (count == 0 || count > 64)
         count = 64;
 
+    if (GetCpuLimit(&cgroupCpuCount))
+        count = cgroupCpuCount;
```

Comment:
The limit should only limit the count, but it should not be able to provide a larger value than the real count of processors if someone sets the period and quota so that their ratio is larger than the real count of processors.